src/constructs/lambda/lambda.ts

```diff
@@ -108,6 +139,21 @@ export class GuLambdaFunction extends Function {
       });
     }
 
+    if (props.config) {
+      // For details, see https://github.com/guardian/ssm-to-env.
```

Comment:
I think that the ssm-to-env project should probably be in the same repo as devx-config which would simplify packaging.

Reply:
Agree - this makes a lot of sense. Hoping to do this in a follow-up but in a way that is transparent to users of @guardian/cdk.